lua/nvim-lsp-installer/servers/wgsl_analyzer/init.lua

```diff
@@ -0,0 +1,32 @@
+local server = require "nvim-lsp-installer.server"
+local cargo = require "nvim-lsp-installer.core.managers.cargo"
+local lspconfig = require "lspconfig"
+local configs = require "lspconfig.configs"
+
+if not configs.wgsl_analyzer then
+    configs.wgsl_analyzer = {
+        default_config = {
+            cmd = { "wgsl_analyzer" },
+            filetypes = { "wgsl" },
+            root_dir = lspconfig.util.root_pattern(".git", "wgsl"),
+            settings = {}
+        }
+    }
+end
```

Comment:
Would you mind opening a PR to lspconfig for adding this server configuration there 🙏? We don't manage any in this plugin